services/object/procs.yml

```diff
@@ -2,6 +2,11 @@ web:
   type: web
   command: node services/object/src/main.js server
   defaultReplicas: 0
+cleanup-expire-objects:
```

Comment:
Judging by the table in `monitoring.mdx`, we usually name this `expire`, so let's do the same here.